consumers/bulksender/main.go

```diff
@@ -149,7 +149,7 @@ func main() {
 
 	config := nsq.NewConfig()
 
-	consumer, err := nsq.NewConsumer(entities.SendBulkTopic, entities.SendBulkTopic, config)
+	consumer, err := nsq.NewConsumer(entities.SenderTopic, entities.SenderTopic, config)
```

Comment:
This should be reversed, since we don't want both consumers to listen to this topic.